wordle-web/app.vue

```diff
@@ -0,0 +1,17 @@
+<template>
+  <NuxtLayout>
+    <v-app >
+      <v-container>
+        <v-btn variant ="text" @click="router.push('/')">Home</v-btn> 
+        <v-btn variant ="text"@click="router.push('/game')">Pentagram</v-btn>
+      </v-container>
+        <NuxtPage />
```

Comment:
Clean implementation of the nav bar, this is a good approach to the navigational needs of the user